.gitignore

```diff
@@ -5,3 +5,4 @@ logs/
 .DS_Store
 bin/
 vendor/
+.idea
```

Comment:
I just add this to my global gitignore - see https://stackoverflow.com/a/7335487/5657035